### PR TITLE
add magisk support

### PR DESCRIPTION
**Pull request by nlifew**
to build a Magisk module, use
$>`gradlew task :app:assembleMagiskRelease`

### CODE REVIEW

File: app/magisk/customize.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+# only for Android 7.0+
```
Comment (MlgmXyysd) [inline app/magisk/customize.sh:2]: There should be more if here. Android 9+ is not supported yet.

File: app/magisk/META-INF/com/google/android/update-binary [hunk 2]
```diff
@@ -0,0 +1,173 @@
+#!/sbin/sh
```
Comment (MlgmXyysd) [inline app/magisk/META-INF/com/google/android/update-binary:1]: Here's an update of Magisk update_binary.

https://github.com/topjohnwu/Magisk/blob/master/scripts/module_installer.sh

**Comment by MlgmXyysd** [comment]
It seems that the current implementation is not suitable for mounting with Magisk, which prevents other software or modules (if any) from modifying public.libraries.txt.

**Comment by canyie** [comment]
我已经改了实现，所以你这个pr用不到了，不过还是很感谢

**Pull request closed without merging**